lucille-core/src/main/java/com/kmwllc/lucille/util/CamelCaseConfigConverter.java

```diff
@@ -0,0 +1,125 @@
+package com.kmwllc.lucille.util;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import java.io.*;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.apache.commons.text.CaseUtils;
+
+public class CamelCaseConfigConverter {
+
+  private static final Logger log = LoggerFactory.getLogger(CamelCaseConfigConverter.class);
+
+  private static String snakeToCamel(String snakeStr) {
+    return CaseUtils.toCamelCase(snakeStr, false, '_');
+  }
+
+  private static void writeToConfFile(String configString, String filePath) throws IOException {
+    try (PrintWriter writer = new PrintWriter(new FileWriter(filePath))) {
+      writer.printf(configString);
+      log.info("Written new config to {}", filePath);
+    } catch (Exception e) {
+      throw new IOException("Error writing new config to file path: " + filePath, e);
+    }
+  }
+
+  private static String applyCamelCase(File file, List<String> stageProperties) throws IOException {
+    StringBuilder sb = new StringBuilder();
+
+    try (BufferedReader br = new BufferedReader(new FileReader(file))) {
```

Comment:
explore whether we can replace this code with the "Simple Code" example in this StackOverflow answer: https://stackoverflow.com/a/40836618

basically, we'd be doing StringUtils.replaceEach( text, keys, values )
using org.apache.commons.lang3.StringUtils

Reply:
added comment in [f90fc30](https://github.com/kmwtechnology/lucille/pull/203/commits/f90fc3031cc2f50349bddcef03f60cea5629a064)